### ISSUE TITLE
Implement Event Archive

### ISSUE THREAD

**Issue by pulsejet** [DevCom-IITB/instiapp-api#35]
Events marked as archived will not show up in the regular feed, but are still accessible through a separate section. ~~Create a separate issue for automatically archiving old events once this is done.~~

**Comment by pulsejet** [comment]
This is a mistake that thankfully didn't happen xD @e-aakash
